autoload/vimlparser.vim

```diff
@@ -3419,7 +3449,8 @@ function! s:ExprParser.parse_expr9()
         endif
       endwhile
     endif
-  elseif token.type == s:TOKEN_COPEN
+  elseif token.type == s:TOKEN_COPEN || token.type == s:TOKEN_RAWCOPEN
+    let is_rawdict = token.type == s:TOKEN_RAWCOPEN
```

Comment:
`is_litdict` (literal dict) might be better?

Reply:
litdict is short and seems reasonable. thanks!